gradle/libs.versions.toml

```diff
@@ -69,7 +71,7 @@ pages = { group = "uk.gov.android", name = "pages", version.ref = "govuk-ui" }
 slf4j-api = { group = "org.slf4j", name = "slf4j-api", version = "2.0.16" }
 theme = { group = "uk.gov.android", name = "theme", version.ref = "govuk-ui" }
 uiautomator = { group = "androidx.test.uiautomator", name = "uiautomator", version = "2.3.0" }
-authentication = "uk.gov.android:authentication:0.5.0"
+authentication = "uk.gov.android.authentication:app:0.6.2"
```

Comment:
Could this be put into the same format as the other libraries i.e.
`authentication = { group = "uk.gov.android.authentication", name = "app", version.ref = "<version_ref>" }`?

Reply:
agreed